code/game/turfs/simulated/minerals.dm

```diff
@@ -521,3 +528,36 @@
 	baseturfs = /turf/open/floor/plating/asteroid/basalt/lava_land_surface
 	initial_gas_mix = LAVALAND_DEFAULT_ATMOS
 	defer_change = 1
+
+/turf/closed/mineral/strong
+	name = "Very strong rock"
+	desc = "Seems to be stronger than the other rocks in the area. Only a master of mining techniques could destroy this."
+	environment_type = "basalt"
+	turf_type = /turf/open/floor/plating/asteroid/basalt/lava_land_surface
+	baseturfs = /turf/open/floor/plating/asteroid/basalt/lava_land_surface
+	initial_gas_mix = LAVALAND_DEFAULT_ATMOS
+	defer_change = 1
+	smooth_icon = 'icons/turf/walls/rock_wall.dmi'
+
+/turf/closed/mineral/strong/attackby(obj/item/I, mob/user, params)
+	if(!ishuman(user))
+		to_chat(usr, "<span class='warning'>Only a more advanced species could break a rock such as this one!</span>")
+		return FALSE
+	var/mob/living/carbon/human/H = user
+	if(H.dna.get_skill_level(SKILL_MINING) == SKILL_LEVEL_LEGENDARY)
+		. = ..()
```

Comment:
Is it a problem that this returns null instead of FALSE if the skill check fails?

Reply:
no

Reply:
Aight that's what I thought, just being sure.